src/apis/friends/index.ts

```
@@ -29,19 +28,19 @@ export const useGetRequestFriendList = () => {
-      const {data} = await instance.get<RequestFriendListType>(`${router}/request`);
+      const { data } = await instance.get<RequestFriendListType>(`${router}/request`)
       return data.friendRequests
-    }
+    },
   })
 }
 
-export const useDeleteFriend = (option:MutationOptions<void, Error, DeleteFriendRequestType>) => {
-  return useMutation<void, Error, DeleteFriendRequestType>({
+export const useDeleteFriend = (option: MutationOptions<void, Error, DeleteFriendRequestType>) => {
+  return useMutation<void, Error, any>({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)